spec/integration_spec.rb

```diff
@@ -2,19 +2,24 @@
 
 describe "Pliny integration test" do
   before(:all) do
-    FileUtils.rm_rf("/tmp/plinytest")
-    FileUtils.mkdir_p("/tmp/plinytest")
-    Dir.chdir("/tmp/plinytest")
+    @original_dir = Dir.pwd
```

Comment:
We need to capture this here, and change back to this dir in the `after(:all)` because Rspec doesn't provide a block mechanism for the `:all` hooks. But it will at least guarantee the `after(:all)` will run and thus CD us back to the right place.

Reply:
👍